backend/data/src/main/kotlin/io/tolgee/security/IsServerAdmin.kt

```diff
@@ -0,0 +1,3 @@
+package io.tolgee.security
+
+annotation class IsServerAdmin()
```

Comment:
I would call this rather "NeedsSeverAdmin" or "AccessWithServerAdminPermission".